tests/php/src/Cli/Export/ExportOptions.php

```diff
@@ -0,0 +1,304 @@
+		'/^wpforms_(activated|version|version_lite)$/',
+		'/^wpseo_(flush_rewrite|ryte)$/',
+		'/^wpsupercache_.*$/',
+	];
+
+	/**
+	 * Associative array of options and their default values for skipping.
+	 *
+	 * @var array
+	 */
+	const OPTION_DEFAULTS = [
+		'use_balanceTags'                 => '0',
+		'use_smilies'                     => '1',
+		'posts_per_rss'                   => '10',
+		'rss_use_excerpt'                 => '0',
+		'posts_per_page'                  => '10',
+		'date_format'                     => 'F j, Y',
+		'time_format'                     => 'g:i a',
+		'category_base'                   => '',
+		'template'                        => 'astra',
+		'stylesheet'                      => 'astra',
+		'use_trackback'                   => '0',
+		'thumbnail_size_w'                => '150',
+		'thumbnail_size_h'                => '150',
+		'thumbnail_crop'                  => '1',
+		'medium_size_w'                   => '300',
+		'medium_size_h'                   => '300',
+		'large_size_w'                    => '1024',
+		'large_size_h'                    => '1024',
+		'thread_comments'                 => '1',
+		'thread_comments_depth'           => '5',
+		'comments_per_page'               => '50',
+		'medium_large_size_w'             => '768',
+		'medium_large_size_h'             => '0',
+		'wp_page_for_privacy_policy'      => '0',
+		'disallowed_keys'                 => '',
+		'auto_plugin_theme_update_emails' => [],
+		'__uagb_do_redirect'              => '',
+	];
+
+	/**
+	 * Process the export step.
+	 *
+	 * @param ExportResult $export_result Export result to adapt.
+	 *
+	 * @return ExportResult Adapted export result.
+	 */
+	public function process( ExportResult $export_result ) {
+		$option_keys = array_filter(
+			$this->get_option_keys(),
+			[ $this, 'skip_excluded_options' ]
+		);
+
+		$options = $this->fetch_options( $option_keys );
+		$options = array_filter(
+			$options,
+			[ $this, 'skip_default_values' ],
+			ARRAY_FILTER_USE_BOTH
+		);
+
+		$options = $this->adapt_options( $options );
+
+		$export_result->add_step( 'import_options', compact( 'options' ) );
+
+		return $export_result;
+	}
+
+	/**
+	 * Get all options stored in the current WordPress instance.
+	 *
+	 * @return string[]
+	 */
+	private function get_option_keys() {
+		global $wpdb;
+
+		return $wpdb->get_col(
+			sprintf( 'SELECT `option_name` FROM `%s`;', $wpdb->options )
+		);
+	}
+
+	/**
+	 * Fetch the options that are meant to be exported.
+	 *
+	 * @param string[] $keys Keys of the options to fetch.
+	 * @return array Associative array of options to export.
+	 */
+	private function fetch_options( $keys ) {
+		global $wpdb;
+
+		if ( empty( $keys ) ) {
+			return [];
+		}
+
+		$query = sprintf(
+			'SELECT `option_name`, `option_value` FROM `%s` WHERE `option_name` IN (%s);',
+			esc_sql( $wpdb->options ),
+			implode(
+				',',
+				array_map(
+					static function ( $key ) {
+						return "'" . esc_sql( $key ) . "'"; },
```

Comment:
Minor point, but why not:

```suggestion
						return $wpdb->prepare( '%s', $key ),
```